base-server/index.d.ts

```diff
@@ -1105,6 +1132,16 @@ export class BaseServer<
    */
   sendAction(action: Action, meta: ServerMeta): Promise<void> | void
 
+  /**
+   * Sets the name of the queue that will be used to process actions of this type.
+   * Can be used for actions which type was not defined
+   * by any {@link Server#type}.
+   *
+   * @param actionType the action type
+   * @param queue the queue name
+   */
+  setQueue(actionType: string, queue: string): void
```

Comment:
Why we want to use this and not `server.type()`?

Reply:
Nope, let’s use `server.type()`. `server.otherType()` is for proxy, they will need a different approach for queue anyway.

Reply:
maybe we should disable queues for proxy? Right now, if I remove `setQueue`. test 'reacts on wrong backend answer' will fail because all actions will by default go to `main` queue, but the test assumes that actions are executed in parallel

Reply:
Can we change this test? We will release it in major update.